### DISCUSSION TITLE
The function of touching and moving the tab bar on Android does not work.

### DISCUSSION THREAD

**Discussion by qlife1146** [react-navigation/react-navigation#11153] [Q&A]
I thought there was a problem with my code, so I used the example as it is, but I can move it to the slide, but I can't move it to the touch. But it works on the iPad.

Anroid

https://user-images.githubusercontent.com/32091837/212285975-77e96565-98ad-4215-ac98-cec3bf5bf0c9.mp4

iPad

https://user-images.githubusercontent.com/32091837/212286018-1d6ca2c1-74c3-4c83-af85-8818a93ebed5.MP4

**Comment by raajnadar** [comment]
Please add a snack demo that reproduces the issue

**Reply by qlife1146** [reply]
https://snack.expo.dev/@qlife1146/lonely-cashew

It works on snack, and it works even after taking qr code with my phone, but it does not move to touch on vscode with my phone. It's very strange.

**Reply by raajnadar** [reply]
Do you mean when running on your hardware device connected to a laptop? It doesn’t work?

It works with snack & when you run on Expo Go from the same hardware device?

**Reply by qlife1146** [reply]
yes.
snack >qr> android device (o).
vscode >qr> android (x).
vscode >qr> ipad (o).

**Reply by raajnadar** [reply]
which android device? And the OS version? Android Version? Please attach more device details in which it is not working

Edit - Did you try on android Emulator?

**Reply by qlife1146** [reply]
oh, sorry.
my device is galaxy note 10+, android 12, one ui 4.1(latest version)
And i did't try on android emulator. I only tried it on the android emulator in the snack.
